tmuxinator.gemspec

```diff
@@ -42,5 +42,14 @@ Gem::Specification.new do |s|
   s.add_dependency "erubis", "~> 2.6"
 
   s.add_development_dependency "bundler", "~> 1.3"
+  s.add_development_dependency "rake", "~> 10.4"
+  s.add_development_dependency "rspec", "~> 3.3"
+  s.add_development_dependency "simplecov", "~> 0.11.0"
+  s.add_development_dependency "coveralls", "~> 0.7"
+  s.add_development_dependency "awesome_print", "~> 1.2"
+  s.add_development_dependency "pry", "~> 0.10"
+  s.add_development_dependency "factory_girl", "~> 4.5"
+  s.add_development_dependency "activesupport", "< 5.0.0"  # Please see issue #432
```

Comment:
Hey there! I'm loving this pull request. My only comment is whether this particular line, adding activesupport as a dependency (granted, only a development dependency) is actually needed.

I'll do a bit of digging to see whether it's needed, and will get back to you.

Reference: #432, #458
Fixes #459